src/main/java/com/matchus/domains/team/domain/Grade.java

```diff
@@ -14,6 +15,8 @@
 	HIRED("용병", 3),
 	;
 
+
+	public static final Set<Grade> VALID_GRADE = Set.of(CAPTAIN, SUB_CAPTAIN);
 	private final String type;
 	private final int order;
```

Comment:
```suggestion

	public static final Set<Grade> VALID_GRADE = Set.of(CAPTAIN, SUB_CAPTAIN);
	
	private final String type;
	private final int order;
```
개행 제거하고 상수랑 멤버변수랑 분리하면 좋을 것 같아요!